kaldb/src/main/java/com/slack/kaldb/writer/SpanFormatter.java

```diff
@@ -196,6 +196,10 @@ public static LogMessage toLogMessage(Trace.Span span) {
         jsonMap.put(key, tag.getVFloat64());
       } else if (valueType == 4) {
         jsonMap.put(key, tag.getVBinary().toStringUtf8());
+      } else if (valueType == 5) {
```

Comment:
We should add some extra asserts in the `SpanFormatterTest` for these types.

Reply:
I'm going to hold off doing anything on this. I feel like we need more tests that do the following - Take Spans and convert them to lucene documents and assert field types

SpanFormatterTest seems to convert Spans to LogMessage which we are getting rid of in the next PR

Reply:
Ah, that makes sense. 👍